src/constants/minions.js

```diff
@@ -25,7 +25,7 @@ export const minion_slots = {
   650: 26,
 };
 
-export const minions_max_uniques = 631;
+export let minions_max_uniques = 0;
```

Comment:
try to avoid `export let` because it looks like it should be assignable but actually isn't